app/controllers/pets_controller.rb

```diff
@@ -31,16 +31,21 @@ def update
       redirect_to "/pets/#{params[:id]}"
     else
       flash[:notice] = "Unsuccessful shelter submission, please fill in the following fields prior to submission: #{missing_fields.each { |field| p "#{field} "}}"
-      redirect_to "/pets/#{params[:id]}/edit"
+      redirect_to "/pets/#{params[:id]}"
```

Comment:
hm, if the form fields aren't completed and the user is wanting to update the pet info, shouldn't it redirect back to or render the edit view? what was the reasoning behind changing this redirect path back to the pet show page?